chainstate/src/interface/chainstate_interface_impl.rs

```diff
@@ -91,6 +91,8 @@ impl<S: BlockchainStorage, V: TransactionVerificationStrategy> ChainstateInterfa
     }
 
     fn is_block_in_main_chain(&self, block_id: &Id<Block>) -> Result<bool, ChainstateError> {
+        // FIXME: should it just delegate to ChainstateRef::is_block_in_main_chain?
```

Comment:
I don't see why not. It's always preferable to centralize these calls for consistency.